pkg/storage/storage.go

```diff
@@ -61,14 +62,27 @@ func New(config *config.Config, linter common.Lint, metrics monitoring.MetricSer
 		)
 	} else {
 		storeName := fmt.Sprintf("%v", config.Storage.StorageDriver["name"])
-		if storeName != constants.S3StorageDriverName {
+		switch storeName {
+		case constants.S3StorageDriverName:
+			// backward compatibility for s3 storage driver
```

Comment:
I think setting the default for `forcepathstyle` should be in the same place as the other tweaks for the configuration:

https://github.com/project-zot/zot/blob/main/pkg/cli/server/root.go#L513

Reply:
@andaaron could it be possible that test are failing not, because the default values from https://github.com/project-zot/zot/blob/main/pkg/cli/server/root.go?rgh-link-date=2024-08-01T06%3A42%3A18Z#L513 are not applied for test?